core/wiki/macros/toc.tid

```diff
@@ -1,218 +1,422 @@
-          <h1><<toc-caption>></h1>
-          <$transclude mode="block">$missingText$</$transclude>
-        </$transclude>
-      </$reveal>
-      <$reveal stateTitle=<<__selectedTiddler__>> type="match" text="">
-        $unselectedText$
-      </$reveal>
-    </div>
-  </div>
+<!-- ==============================================================
+Shows a tabbed toc.
+If a toc-item link is clicked it will open the tiddler in the story 
+=================================================================== -->
+\procedure toc-tabbed-external-nav(tag,sort:"",selectedTiddler:"$:/temp/toc/selectedTiddler",unselectedText,missingText,template:"",exclude,captionField,tagField,parentField,itemTemplate)
+<$tiddler tiddler={{{ [<selectedTiddler>get[text]] }}}>
+	<div class="tc-tabbed-table-of-contents">
+		<$linkcatcher to=<<selectedTiddler>>>
+			<div class="tc-table-of-contents">
+				<!-- v5.3.4 itemClassFilter has been replaced by tf.toc-itemClassFilter -->
+				<$macrocall $name="toc-selective-expandable"
+					tag=<<tag>> sort=<<sort>> exclude=<<exclude>>
+					captionField=<<captionField>> tagField=<<tagField>> parentField=<<parentField>>
+					itemTemplate=<<itemTemplate>>
+				/>
+			</div>
+		</$linkcatcher>
+		<div class="tc-tabbed-table-of-contents-content">
+			<$reveal stateTitle=<<selectedTiddler>> type="nomatch" text="">
+				<$transclude mode="block" tiddler=<<template>>>
+					<h1><<_itemTemplate>></h1>
+					<$transclude mode="block">$missingText$</$transclude>
+				</$transclude>
+			</$reveal>
+			<$reveal stateTitle=<<selectedTiddler>> type="match" text="">
+				<<unselectedText>>
+			</$reveal>
+		</div>
+	</div>
 </$tiddler>
 \end
 
-\define toc-tabbed-internal-nav(tag,sort:"",selectedTiddler:"$:/temp/toc/selectedTiddler",unselectedText,missingText,template:"",exclude)
-\whitespace trim
-<$linkcatcher to=<<__selectedTiddler__>>>
-  <$macrocall $name="toc-tabbed-external-nav" tag=<<__tag__>> sort=<<__sort__>> selectedTiddler=<<__selectedTiddler__>> unselectedText=<<__unselectedText__>> missingText=<<__missingText__>> template=<<__template__>> exclude=<<__exclude__>> />
+<!-- ========================================================
+Shows a tabbed toc. If a toc-item link is clicked 
+it will open the tiddler in the view area of the same tiddler
+============================================================= -->
+\procedure toc-tabbed-internal-nav(tag,sort:"",selectedTiddler:"$:/temp/toc/selectedTiddler",unselectedText,missingText,template:"",exclude,captionField,tagField,parentField,itemTemplate)
+<$linkcatcher to=<<selectedTiddler>>>
+	<$macrocall $name="toc-tabbed-external-nav"
+		tag=<<tag>>
+		sort=<<sort>>
+		selectedTiddler=<<selectedTiddler>>
+		unselectedText=<<unselectedText>>
+		missingText=<<missingText>>
+		template=<<template>>
+		exclude=<<exclude>>
+		captionField=<<captionField>>
+		tagField=<<tagField>>
+		parentField=<<parentField>>
+		itemTemplate=<<itemTemplate>>
+	/>
```

Comment:
This file seems like it is now unmanageably large.

Reply:
I did reply to this one, in the main thread